### ISSUE TITLE
Allow to set String as badgeValue

### ISSUE THREAD

**Issue by iad24** [johncosch/TIPBadgeManager#6]

**Comment by johncosch** [comment]
@iad24 thanks for the feature request! I have thought about this kind of functionality, but i'm not sure about how well it fits the project. The purpose of this project is to allow for the easy creation of badges and for synchronization across badges. The issue with implementing this appears in the latter, because the application badge cannot be set as anything other than an Int. This could lead to some ambiguity as the synchronizing would no longer work across all badges. I'll keep this issue open while I continue to explore this feature. Thanks again!

**Comment by iad24** [comment]
Yes, usually badges are Ints but for flexibility-wise it would be nice to be a String (meaning you can set anything into it) =)

In my case, my app's requirement is if the badge value set is greater than 99, we put "99+".

If you got spare time please add this feature. Nice work you got there though!

**Comment by johncosch** [comment]
Ohh ok, that's a really good use case for strings that I haven't really considered. It actually shouldn't be that hard to do because both TIPBadgeView and the UITabBar badge value can be set as Strings. Again I think the real issue is doing this while maintaining the clarity of the api. I'll be working on this more in the coming days. Thanks for the compliments!
